### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HuckMatt**
Update README to new format.

### CODE REVIEW

**Comment by HuckMatt** [comment]
@eric-giannini please review

File: README.md [hunk 1]
```diff
@@ -1,18 +1,40 @@
-# Symbl Video React App
+# Symbl & Twilio Video React App
 
-This is a multi-party video conferencing application that demonstrates [Symbl's Real-time APIs](https://docs.symbl.ai/#real-time-websocket-api). This application is inspired by [Twilio's video app](https://github.com/twilio/twilio-video-app-react) and is built using [twilio-video.js](https://github.com/twilio/twilio-video-app-react) and [Create React App](https://github.com/facebook/create-react-app).
```
Comment (eric-giannini) [inline README.md:3]: Please add the description in the introduction.

File: README.md [hunk 2]
```diff
@@ -111,5 +127,46 @@ If you want to see how the application behaves with multiple participants, you c
...
+    "twilio-video": "^2.7.1"
+  }
+```
+
+## Conclusion
+When implemented this application will allow you to join a demo twilio meeting, and Symbl transcripts will be displayed on screen in real time. 
+
+## Community
+
+If you liked our integration guide, please star our repo! If you have any questions, feel free to reach out to us at devrelations@symbl.ai or through our Community Slack at https://developer.symbl.ai/community/slack or our [developer community][developer_community].
```
Comment (eric-giannini) [inline README.md:167]: Please reformat.

**Comment by HuckMatt** [comment]
@eric-giannini updated updated.

**Comment by eric-giannini** [comment]
@HuckMatt The beginning of the README is not rendering properly. Check out the screenshot: 
Screen Shot 2021-02-01 at 1 28 44 PM

**Pull request merged**